src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $calendar-width;
+  gap: $calendar-day-gap;
+
+  .calendar__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    width: $calendar-day-size;
+    height: $calendar-day-size;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
```

Comment:
It's recommended to store font sizes in variables, especially when they are used multiple times in the project. This will make your code more maintainable and consistent.